Tests/SPIManifestTests/ManifestTests.swift

```diff
@@ -58,7 +62,37 @@ class ManifestTests: XCTestCase {
 
         // validation
         XCTAssertEqual(m,
-                       Manifest(builder: .init(configs: [
+                       Manifest(metadata: .init(authors: "Christian Noon, Jon Shier and other contributors"),
+                                builder: .init(configs: [
+                        .init(platform: Platform.watchos.rawValue,
+                              scheme: "Alamofire watchOS")
+                       ]))
+        )
+    }
+
+    func test_load_without_authors() throws {
```

Comment:
I'm not 100% sure what this test is intended to test. The other tests that load manifests without `metadata` or `authors` test that the whole key can be omitted. Is the purpose of this test to ensure that we get `nil` if people add the keys but without data?

If that's the case maybe `test_load_nilAuthors` might be a better name? I'm not sure we need to test this case though.

Reply:
you are right. I think we do not to need this test